packages/jest-cli/src/watch.js

```diff
@@ -195,8 +201,14 @@ const watch = (
         );
         break;
       case KEYS.QUESTION_MARK:
-        if (process.env.JEST_HIDE_USAGE) {
+        break;
```

Comment:
Oh, that's right, I've missed that. I think we should get rid of the question mark.